Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -69,6 +75,46 @@ public void AfterTest()
             var testName = $"{TestContext.CurrentContext.Test.FullName}";
 
             TestEnvironment.RecordTestPerformance(testName, _stopwatch.Elapsed);
+            RemoveTables();
+        }
+
+        private void RemoveTables()
+        {
+            if (_tablesToRemove.Count == 0)
+                return;
+            
+            using (var conn = new SnowflakeDbConnection(ConnectionString))
+            {
+                conn.Open();
+
+                var cmd = conn.CreateCommand();
+
+                foreach (var table in _tablesToRemove)
+                {
+                    cmd.CommandText = $"DROP TABLE IF EXISTS {table}";
+                    cmd.ExecuteNonQuery();
+                }
+            }
+        }
+
+        protected void CreateOrReplaceTable(string tableName, IEnumerable<string> columns, string additionalQueryStr = null)
+        {
+            var columnsStr = string.Join(", ", columns);
+            
+            using (var conn = new SnowflakeDbConnection(ConnectionString))

```

Comment:
because in the derived test this ConnectionString might get changed for some reasons